### PR TITLE
Added a test for subdivision vertex and face count.

### PR DESCRIPTION
**Pull request by anderejd**
Added to catch similar bugs as found in #2

### CODE REVIEW

File: tests/integration_tests.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/// Test added for https://github.com/huxingyi/meshlite/pull/2
+#[test]
+fn verify_cube_subdivision() {
+    let mut mesh = cube();
+    let sub = mesh.subdivide();
+    assert_eq!(26, sub.vertex_count);
+    assert_eq!(24, sub.face_count);
+}
+
+/// Should this be part of the public API? (currently defined in two other places)
```
Comment (huxingyi) [inline tests/integration_tests.rs:15]: I also think making it public is better, it's a common API. All current public APIs are for Dust3D, if we want meshlite to be a general library, there are much more APIs should be added.

Comment (anderejd) [reply tests/integration_tests.rs:15]: Sounds good. Would you like to think about how to structure that some more or should I make a PR?

Comment (huxingyi) [reply tests/integration_tests.rs:15]: How about add to a separate file such as premitives.rs?

Comment (anderejd) [reply tests/integration_tests.rs:15]: Sounds like a good idea.

**Pull request merged**